Framework/DataObjects/inc/MantidDataObjects/TableColumn.h

```diff
@@ -282,8 +287,8 @@ inline bool TableColumn<std::string>::compareVectors(const std::vector<std::stri
 
 /// Template specialisation for strings for comparison
 template <>
-inline bool TableColumn<API::Boolean>::compareVectors(const std::vector<API::Boolean> &newVector,
-                                                      double tolerance) const {
+inline bool TableColumn<API::Boolean>::compareVectors(const std::vector<API::Boolean> &newVector, double tolerance,
+                                                      bool const) const {
   (void)tolerance;
```

Comment:
This isn't in your change already, but we have a macro `UNUSED_ARG` defined in `DllConfig.h` to use. The code would be
```cpp
UNUSED_ARG(tolerance);
```

Reply:
Just figured out what this meant.  I didn't see the `(void)tolerance;` It's actually possible to remove the name `tolerance` from the argument list entirely and not require either `(void)` or `UNUSED_ARG`